docs/src/modules/components/DemoEditor.tsx

```diff
@@ -27,6 +27,9 @@ const StyledMarkdownElement = styled(MarkdownElement)(({ theme }) => ({
         theme.palette.mode === 'dark' ? theme.palette.primaryDark.main : theme.palette.primary.main
       }`,
     },
+    [theme.breakpoints.up('sm')]: {
+      borderRadius: theme.shape.borderRadius,
+    },
```

Comment:
Fix code preview that look strange on mobile

Before:

<img width="409" alt="Screenshot 2022-11-12 at 17 30 33" src="https://user-images.githubusercontent.com/3165635/201484302-0984982d-3e1b-4c9e-a22e-0afbc3d9feff.png">

https://mui.com/material-ui/react-skeleton/

After:

<img width="395" alt="Screenshot 2022-11-12 at 17 31 22" src="https://user-images.githubusercontent.com/3165635/201484326-99a6a379-bd28-43e4-8337-1ea1b7c24e44.png">

https://deploy-preview-35108--material-ui.netlify.app/material-ui/react-skeleton/